cmd/create/cluster/cmd.go

```diff
@@ -598,6 +611,20 @@ func run(cmd *cobra.Command, _ []string) {
 		isIAM = !isSTS
 	}
 
+	isHostedCluster := args.hostedClusterEnabled
+	if interactive.Enabled() {
```

Comment:
Since we want to hide the flag until it's ready, we can use something like:
```suggestion
	if if cmd.Flags().Changed("hosted-cp") && interactive.Enabled() {
```
to enable showing it in interactive mode for testing and internal usage without it being presented to users yet.

Reply:
Great catch! I almost forgot that this is a hidden flag. Will fix this.